### PR TITLE
update some Permalinks to RelPermalinks

### PR DESCRIPTION
**Pull request by FlyingOnion**
If we build our own sites, PermaLink will still link to kubernetes.io

### CODE REVIEW

**Comment by FlyingOnion** [comment]
@stewart-yu 能帮忙看一下吗？已经过一段时间了还没人来

File: layouts/blog/baseof.html [hunk 1]
```diff
@@ -20,15 +20,15 @@ <h1>{{ .Title }}</h1>
...
+						<a class="widget-link" href="/docs/contribute/start/#write-a-blog-post"><i class="fa fa-edit fab-icon"></i> Submit a Post</a>
 						<a class="widget-link" href="https://twitter.com/kubernetesio"> <i class="fab fa-twitter-square fab-icon"> </i> @Kubernetesio</a>
 						<a class="widget-link" href="https://github.com/kubernetes/kubernetes"> <i class="fab fa-github-square fab-icon"></i> View on GitHub </a>
 						<a class="widget-link" href="http://slack.k8s.io"><i class="fab fa-slack fab-icon"> </i> #kubernetes-users </a>
 						<a class="widget-link" href="https://stackoverflow.com/questions/tagged/kubernetes"><i class="fab fa-stack-overflow fab-icon"></i> Stack Overflow</a>
 						<a class="widget-link" href="https://discuss.kubernetes.io"><i class="fab fa-discourse fab-icon"></i>Forum</a>
-						<a class="widget-link" href="https://kubernetes.io/docs/setup/pick-right-solution/"><i class="fa fa-download fab-icon"></i> Download Kubernetes</a>
+						<a class="widget-link" href="/docs/setup/pick-right-solution/"><i class="fa fa-download fab-icon"></i> Download Kubernetes</a>
```
Comment (jimangel) [inline layouts/blog/baseof.html]: should be: `/docs/setup/` (https://kubernetes.io/docs/setup/pick-right-solution/ was moved)

**Comment by kbarnard10** [comment]
@FlyingOnion Could you make the requested change and resolve merge conflicts? Then I'll get this merged.

**Comment by xiangpengzhao** [comment]
@FlyingOnion sorry for late response. Are you still working on this? If so, please rebase the PR and then we can merge it. Thanks for contribution!

**Comment by FlyingOnion** [comment]
> @FlyingOnion sorry for late response. Are you still working on this? If so, please rebase the PR and then we can merge it. Thanks for contribution!

I will get this done next week

**Comment by FlyingOnion** [comment]
Seems making a mess after resolving the conflict. I will open a new pull request.

**Pull request closed without merging**